tests/test_downloadermiddleware_httpcache.py

```diff
@@ -156,7 +160,12 @@ def _get_settings(self, **new_settings):
 
 class LeveldbStorageTest(DefaultStorageTest):
 
-    pytest.importorskip('leveldb')
+    try:
+        pytest.importorskip('leveldb')
+    except SystemError:
+        # Happens in python 3.8
+        # This will cause xfail in DefaultStorageTest to trigger
+        pass
```

Comment:
I think you can use `pytest.skip("Cannot import leveldb")` here (raises an exception), and not have to use `@pytest.mark.xfail` above.

Reply:
Yeah, that should work. When I'm back at my computer later I'll remove those xfails and use the pytest.skip here instead.